x/interchainqueries/client/cli/query.go

```diff
@@ -112,3 +113,27 @@ func CmdQueryRegisteredQueryResult() *cobra.Command {
 
 	return cmd
 }
+
+func CmdQueryLastRemoteHeight() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "query-last-remote-height [connection-id]",
+		Short: "queries last remote height by connection id",
```

Comment:
as far as I can see, there is no way to get the remote height but via using connection id, is there?

Reply:
you may have many networks connected so you have to identify one by connection id